packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -59,6 +59,12 @@ class BottomSheetRangeCell extends Component {
 		AppState.removeEventListener( 'change', this.handleChangePixelRatio );
 	}
 
+	toFixed( num ) {
```

Comment:
I think that most of the changes in that file could be a part of separate PR where you add `toFixed` prop to the range-cell. This columns PR should contain only the part with `rangePreview`

Reply:
Moved range cell changes into https://github.com/WordPress/gutenberg/pull/24379